### PR TITLE
Update organizing team

### PR DESCRIPTION
**Pull request by glemaitre**
Updating the organizing team as suggested in a previous meeting.

### CODE REVIEW

File: content/pages/2023/about.md [hunk 1]
```diff
@@ -7,72 +7,25 @@ template: page
...
-</br>
-</br>
-
-#### Sprints chairs
+### Organising team (in alphabetical order)
 
+- Alessia Marcolini (LUMICKS, NL)
+- Alexander CS Hendorf (Königsweg GmbH, DE)
+- Dr. Adrin Jalali (DE)
+- Artem Kislovskiy (Verisign, CH)
```
Comment (Kislovskiy) [inline content/pages/2023/about.md:15]:
```suggestion
- Artem Kislovskiy (CH)
```

**Review by Kislovskiy — COMMENTED** [review]
Mike shared with me the following emails `@euroscipy.org`, maybe we can include the link to a few:

chair
grants
info
maintainers
noreply
program
sponsoring
students
tutorials
webmaster

**Comment by glemaitre** [comment]
We can put the address in the contact us: https://www.euroscipy.org/2023/contact_us.html

**Comment by glemaitre** [comment]
I think that we are covered with the current list of emails by checking the "Contact Us" page. WDYT @Kislovskiy ?

**Comment by Kislovskiy** [comment]
> I think that we are covered with the current list of emails by checking the "Contact Us" page. WDYT @Kislovskiy ?

Looks good to me. I just remember someone was mentioning that he expects to see the contacts of the organizers on the same page. I'm fine to place them on the separate page.

**Comment by glemaitre** [comment]
> Looks good to me. I just remember someone was mentioning that he expects to see the contacts of the organizers on the same page. I'm fine to place them on the separate page.

Let's put a link to redirect the contact us at the end of the page then. It makes sense.

**Pull request merged**